### PR TITLE
Next

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Merge current into next

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/dbt-versions/2024-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+title: "2024 dbt Cloud release notes"
+description: "Weekly release notes for dbt Cloud 2024"
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/2024-release-notes.md]:
```suggestion
description: "dbt Cloud release notes for 2024"
```

**Comment by mirnawong1** [comment]
hey @matthewshaver , great job and thanks so much for consolidating this! I've got some feedback on the layout:

- structure &mdash; maybe we can have a more consistent, clean layout that makes it easy to distinguish between different types of updates (new features, enhancements, fixes). i think if we use headers and subheaders, it might allow us to bulk related topics inside the header (e.g. all the 'new' goes under the `## new` header, etc. i also wonder if we can consider adding icons or colo to visually separate different sections?
- expandable &mdash;  maybe we can indent the expandable so its in line with the bullets. currently it's looking all weird. i also wonder if we can have the expendables grouped grouped together so it doesn't interrupt the bullet bullet bullet,etc flow?  
- right toc &mdash; might be good to have a right toc so users can quickly navigate?
- intro &mdash; nice-to-have a more fleshed-out intro sentence?

# example

## April 2024

### What’s new:
- Merge jobs [Beta] - Trigger dbt job runs via Git pull requests....
- Semantic Layer enhancements - Additional support for Privatelink and SSO....

### Enhancements:
- Select entities for group by, filter by, and order by in Semantic Layer...,

### Fixes:

- Resolved dbt parse error when using list filters.
- Improved join_to_timespine application in metrics.

**Comment by mirnawong1** [comment]
also adding replicated's docs as inspo: https://docs.replicated.com/release-notes/rn-app-manager

**Comment by matthewshaver** [comment]
@mirnawong1 Great feedback and good ideas worth considering. We currently have an Asana task open to fix the formatting errors for the lifecycle pills so we can use those in front of the items. 

As for the grouping, we need to take into consideration that these will be grouped by week once we go live (based on customer requests for release notes). We've taken the historical data and grouped it by Month but when we go live it will look more like this:

## MONTH
### WEEK 1
- **New:** Feature
- **Fix:** Feature
- **Enhancement:** Feature

### WEEK 2
- **New:** Feature
- **Fix:** Feature
- **Enhancement:** Feature

and so on

Let's bring it to the group when I get back and figure out what we think is the best way to group in that context.

Right TOC - will be adding that, but the automation removes it from that section, so we have to figure out a way to put it back. If you have ideas, I'm open to them. 

Intro - it's a pretty straightforward section, so I kept it simple, but I'm open to any additional language you think would be good. 

Expandables - I'll try the indenting and see what that looks like now! Great idea. I also have an Asana task in to get those to support markdown formatting in the title

File: website/docs/docs/dbt-versions/2024-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+description: "dbt Cloud release notes for 2024"
+id: "2024-release-notes"
+sidebar: "2024 dbt Cloud release notes"
+pagination_next: null
+pagination_prev: null
+---
+
+dbt Cloud release notes for 2024.
+
+# April 2024
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/2024-release-notes.md:12]: @matthewshaver If you change these to an H2 they show up in the right hand TOC!
```suggestion
## April 2024
```

**Review by runleonarun — APPROVED** [review]
@matthewshaver small suggestion for right ToC

**Comment by runleonarun** [comment]
@mirnawong1 and @matthewshaver I added some thoughts about layout in the [follow-up issue](https://github.com/dbt-labs/docs.getdbt.com/issues/5377)

Quick question: when will we remove the old jan through April folders and add redirects?

**Comment by matthewshaver** [comment]
> @mirnawong1 and @matthewshaver I added some thoughts about layout in the [follow-up issue](https://github.com/dbt-labs/docs.getdbt.com/issues/5377)
> 
> Quick question: when will we remove the old jan through April folders and add redirects?

Yup! I'll work on it when I get back!

**Pull request merged**